### DISCUSSION TITLE
怎么把表单设计器集成到vue项目中

### DISCUSSION THREAD

**Discussion by TanTanming** [alibaba/formily#3852] [Q&A 2.x]
最近想研究一下表单设计器，但是按照文档操作，不知道怎么把设计器集成到 vue 的项目中去，只想搞个 demo 的项目即可，求助求助！！！，我用这个源码 启动后也看不见表单设计器，救救孩子把

**Comment by yanhao98** [comment]
https://github.com/qq1037305420/element-plus

**Reply by TanTanming** [reply]
> https://github.com/qq1037305420/element-plus

clone这个项目，启动项目就能看到表单设计器吗，求教！

**Reply by JetaimeJetadore** [reply]
请问下这个vue2支持吗

**Comment by moluoxixi** [comment]
https://github.com/moluoxixi/fomily-element-plus.git 我重构了一下
